contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,259 @@
+
+- **Re-engage your users:** Users who like your product can sometimes be distracted by other things. Well-timed outreach messages, such as showcasing an underused featured, can be extremely powerful. [Path analysis](/product/user-paths) is a useful tool for understanding where users get distracted.
+
+Tactics for reducing churn is an article in itself, but these are all good starting points. Just remember each lever you pull to reduce churn compound, so it's important to pull more than one if you can. It doesn't matter so much which one works the best, just that one or more them do.
+
+### The things you need to improve retention
+
+You're gonna need a few things to track, analyze and reduce customer churn, such as:
+
+1. The ability to identify your new/current users (e.g. job role, location etc.)
+
+1. An analytics tool to help you track user churn, and analyze use behavior.
+
+1. Session recording, so you can watch real interactions with your product.
+
+1. A way to test new features and improvements to validate if they work.
+
+1. The ability to safely roll out new features to test groups and specific cohorts
+
+1. Product-minded engineers or product managers who talk to users and ship fast
+
+Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.
+
+And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.
+
+Check out [our product page](/product) if this sounds interesting to you.
+
+/pitch over
+
+## Takeaways
+
+Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.
+
+To conclude, here are five takeaways:
+
+1. Higher churn in an early-stage startup is fine, so long as you're growing fast and finding impactful ways to improve your product. Declining churn is a good sign you either have, or are approaching, product-market fit.
+
+1. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than an enterprise SaaS with an ARPC of $10,000 per month.
+
+1. Use cohort retention tables to understand what types of users, or features, positively or negatively impact churn. This will help guide product development, marketing, and areas to experiment in.
```

Comment:
```suggestion
1. Use cohort retention tables to understand what types of users, or features, positively or negatively impact churn. This will help guide product development and marketing.
```